src/main.rs

```diff
@@ -28,6 +28,10 @@ struct Options {
     )]
     /// Set the target branch
     branch: String,
+
+    #[structopt(short = "t", long = "cargo-toml")]
```

Comment:
[structopt seems to allow to pass a default value](https://docs.rs/structopt/0.3.5/structopt/?search=#how-to-derivestructopt)

Maybe the cargo_toml path could be a String, thanks to this feature?

Reply:
That's true. It makes more sense to have it as a `default_value` rather than an `Option<String>`